pm_nq/pm_nq.xml

```diff
@@ -0,0 +1,232 @@
+                version = {v0.2.1},
+                month = {2},
+                year = {2022},
+                doi = {}
+            }
+        </token>
+        <import>muon_macros.xml</import>
+    </macros>
+    <creator>
+        <person givenName="Patrick" familyName="Austin" url="https://github.com/patrick-austin"/>
+        <organization url="https://muon-spectroscopy-computational-project.github.io/index.html" name="The Muon Spectroscopy Computational Project"/>
+    </creator>
+    <requirements>
+        <requirement type="package" version="@TOOL_VERSION@">pymuonsuite</requirement>
+        <requirement type="package" version="1.0.0">euphonic</requirement>
+        <requirement type="package" version="3.0">zip</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        structure_name=\$(sed 's/ //g' <<< '$structure.name') &&
+        #if $task.task == 'w':
+            out_folder=\$(sed 's/.$structure.extension/_displaced/g' <<< '$structure.name') &&
+            ln -s '$structure' \$structure_name &&
+            #if $task.phonon.phonon_source_type == "castep":
+                phonon_source_file_name=\$(sed 's/ //g' <<< '$task.phonon.phonon_source_castep.name') &&
+                ln -s '$task.phonon.phonon_source_castep' \$phonon_source_file_name &&
+                castep_file_name=\$(sed 's/ //g' <<< '$task.phonon.castep_file.name') &&
+                ln -s '$task.phonon.castep_file' \$castep_file_name &&
+            #else if $task.phonon.phonon_source_type == "dftb+":
+                phonon_source_file_name=\$(sed 's/ //g' <<< '$task.phonon.phonon_source_dftb.name') &&
+                ln -s '$task.phonon.phonon_source_dftb' \$phonon_source_file_name &&
+                gen_file_name=\$(sed 's/ //g' <<< '$task.phonon.gen_file.name') &&
+                ln -s '$task.phonon.gen_file' \$gen_file_name &&
+            #end if
+            touch params.yaml &&
+            printf "phonon_source_file: \$phonon_source_file_name\n">>params.yaml &&
+            printf 'phonon_source_type: $task.phonon.phonon_source_type\n'>>params.yaml &&
+            #if $task.script_file:
+                script_file_name=\$(sed 's/ //g' <<< '$task.script_file.name') &&
+                ln -s '$task.script_file' \$script_file_name &&
+                printf "script_file: \$script_file_name\n">>params.yaml &&
+            #end if
+            printf 'method: $task.method\n'>>params.yaml &&
+            printf 'mu_symbol: $task.mu_symbol\n'>>params.yaml &&
+            printf 'mu_index: $task.mu_index\n'>>params.yaml &&
+            printf 'grid_n: $task.grid_n\n'>>params.yaml &&
+            printf 'sigma_n: $task.sigma_n\n'>>params.yaml &&
+            (printf 'k_points_grid: $task.k_points_grid\n' | sed "s/__ob__/[/g" | sed "s/__cb__/]/g") >>params.yaml &&
+            printf 'avgprop: $task.avgprop\n'>>params.yaml &&
```

Comment:
Would be a config-file https://docs.galaxyproject.org/en/latest/dev/schema.html#tool-configfiles-configfile here be better?